sandbox-web/src/main/java/com/exadel/sandbox/team5/ReviewRestController.java

```diff
@@ -3,12 +3,9 @@
 import com.exadel.sandbox.team5.entity.Review;
 import com.exadel.sandbox.team5.service.ReviewService;
 import lombok.RequiredArgsConstructor;
-import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
-import org.springframework.web.server.ResponseStatusException;
 
 import java.util.List;
-import java.util.stream.Collectors;
 
 @RestController
 @RequestMapping("/review")
```

Comment:
Let's use the plural for `review` as well `/review`-> `/reviews` and tell about that frontend team.

Reply:
Done.